GOVERNANCE.md

```diff
@@ -61,19 +61,67 @@ maintainer has already discussed the candidacy with the candidate and a
+the Golang implementations of the cgroups, runc, and console APIs, the ttrpc
+implementation, and various plugins like the CRI implementation. These, among a
+handful of other repositories, are all **core** and required pieces which culminate
+in officially supported and maintained releases of containerd.
+
+Non-core projects have strong affiliation with the containerd runtime, but
+operate in some ways similar to the traditional `contrib/` directory in many
+open source projects. They are most likely separately maintained, and potentially
+have a different release cycle and support guarantee than the main containerd project
+and its core subprojects. They may be written in other languages and therefore
+require different skills, developer tools, and CI systems than the core projects.
+For these reasons, non-core subprojects will be treated differently than core
+projects within the governance of the containerd organization.
+
+Both core and non-core subprojects must adhere to the CNCF
+[charter](https://www.cncf.io/about/charter/) and mission.
+
+### Adding core subprojects
+
+New core subprojects can request to be added to the containerd GitHub
+organization by submitting a project proposal via public forum (a
+`containerd/project` GitHub issue is the easiest way to provide this proposal).
+The existing maintainers are given five business days to discuss the new
 project, raise objections and cast their vote. Projects must be approved by at
 least 66% of the current maintainers by adding their vote.
 
 If a project is approved, a maintainer will add the project to the containerd
 GitHub organization, and make an announcement on a public forum.
 
+To make this transition to being a core subproject easier, please prepare your
+repository with the DCO signoff checks, the fileheader check, and by adding the
+appropriate Apache 2 license file to your project root as well as the boilerplate
+header to all applicable source files.
+
+### Adding non-core subprojects
+
+Non-core subprojects will also submit a project proposal via public forum, and
+should state that the project is expected to be **non-core**.
+
+The proposal should include a proposed list of maintainers who will manage
+the non-core project and provide general information on support, releases,
+stability, and any additional detail useful for the containerd maintainers to
+understand the scope and nature of the project.
+
+The existing maintainers are given five business days to discuss the new
+project, raise objections and cast their vote. Projects must be approved by at
+least 50% of the current maintainers by adding their vote.
+
+If a project is approved, a maintainer will add the project to the containerd
+GitHub organization and provide write access for that repository to the proposed
+maintainer list, as well as make an announcement on a public forum.
+
+Unlike core projects, non-core project maintainers can become reviewers and
```

Comment:
The wording in this paragraph is somewhat unclear. I think we should clarify here that maintainers of non-core projects are only maintainers of that subproject and not maintainers of any other containerd projects, as such, they are not required to participate in project level voting. On the other hand, containerd maintainers listed in this repository are responsible for maintaining all containerd projects, core and non-core. As such, they will have write access to all sub-projects and there is no need for containerd maintainers to be listed as maintainers of non-core projects.

Reply:
Clarified.